VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/participant-waiting-room/private-consultation-participants/participant-item/participant-item.component.html

```diff
@@ -16,20 +16,20 @@
             aria-hidden="true"></span>
     </div>
     <div [class]="getRowClasses(participant)">
-        <span [class]="participantNameClass(participant)">{{participant.display_name}}</span>        
+        <span [class]="'wrap-anywhere ' + participantNameClass(participant)">{{participant.display_name}}</span>
```

Comment:
I have multiple issues with this using a method to set a class of color name. I realise this is existing functionality though so for now lets use ngClass instead for the participantNameClass(participant). Try this and let me know if it doesn't work.

```suggestion
        <span class="wrap-anywhere" [ngClass]="participantNameClass(participant)">{{participant.display_name}}</span>
```